common/metrics/metric_defs.go

```diff
@@ -279,6 +279,14 @@ const (
 	PersistenceListNamespacesScope = "ListNamespaces"
 	// PersistenceGetMetadataScope tracks DeleteNamespaceByName calls made by service to persistence layer
 	PersistenceGetMetadataScope = "GetMetadata"
+	// PersistenceGetNexusServiceScope tracks GetNexusService calls made by service to persistence layer
+	PersistenceGetNexusServiceScope = "GetNexusService"
+	// PersistenceListNexusServicesScope tracks ListNexusService calls made by service to persistence layer
+	PersistenceListNexusServicesScope = "ListNexusServices"
+	// PersistenceCreateOrUpdateNexusServiceScope tracks CreateOrUpdateNexusService calls made by service to persistence layer
+	PersistenceCreateOrUpdateNexusServiceScope = "CreateOrUpdateNexusService"
+	// PersistenceDeleteNexusServiceScope tracks DeleteNexusService calls made by service to persistence layer
+	PersistenceDeleteNexusServiceScope = "DeleteNexusService"
```

Comment:
I know I named the APIs, but I'm considering adding `Incoming` to all of these names to 1. match the operator service API and 2. avoid conflicts with a concept that's coming later of an outgoing service registry.

Reply:
You expect the outgoing services to be handled differently? I.E. we don't expect to be able to reuse these APIs for outgoing services as well?